### DISCUSSION TITLE
Cannot match, i.e. Falcon Studios titles with GEVI

### DISCUSSION THREAD

**Discussion by dimmitri9** [CodyBerenson/PGMA-Modernized#235] [Q&A] [answered]
Good evening, PGMA team! Dimítri, here!

PMS:
Windows 11, 
Intel(R) Core(TM) i3-9100F CPU @ 3.60GHz   3.60 GHz,
8,00 GB RAM, 64bits.

Logs attached:
[com.plexapp.agents.GEVI.log](https://github.com/CodyBerenson/PGMA-Modernized/files/11327909/com.plexapp.agents.GEVI.log)

Well, I'm trying hard to match the files, that are already renamed as the magnificent tutorial says to do
(Studio) - Movie Title (Year) 
PMS restarted, with all the bundle files within the Plugin folder, GEVI box checked at both GayAdult and GayAdultFilms, and a new Plex Movie Library just for the +18 collection, a Falcon folder, specifically.
Within that folder it just has Out of Athens movie, part 1 and part 2

It's not like I laready matched previous titles from other studios, it's my first time trying to match a +18 title with Plex and that brillant PGMA that i came across, as thousand of others users happily found too

Here's the link from GEVI 
https://gayeroticvideoindex.com/video/17395
GEVI is my favorite gay index site, they are awesome

Once again, thank you so much for bringing that feature to so many users!

**Comment by JPH71** [comment]
How did you name the files exactly...
Without this information it will be hard to pin point the issue.

Refresh the metadata for the film and then send me the GEVI log...

Jason x

**Comment by JPH71** [comment]
As you can see Out of Athens scrapes OK
I named it as: (Falcon Studios) - Out of Athens 1 (2000)
![image](https://user-images.githubusercontent.com/43102249/234641123-df85e977-76a9-4ada-b211-26b7bad857f1.png)

Jason.....

**Comment by dimmitri9** [comment]
After a few tries, today I still can't go trought it all
Here's what I'm trying to match right now (I will still tell you about the Out of Athens Part 1 file):
It's a scene (I know, if a I can't seem to match a movie then a scene must be harder):

https://www.waybig.com/blog/2011/08/06/lucasentertainment-backdoor-jonathan-agassi-gabriel-lenfant/
I went to "Agent", set within Gay Adults Scenes Agent just WayBig(IAD) checked, then started a new Movie Library for just that scene
The file is named as: (LucasEntertainment) - Backdoor (Jonathan Agassi & Gabriel Lenfant) (2011)
![Screenshot  2023-04-27 190342](https://user-images.githubusercontent.com/20586368/235008857-a0bc213b-dac0-4fd8-aa34-5d445ef33cdd.jpg)
Here's what it shows and how it reads the file
![Screenshot 2023-04-27 190233](https://user-images.githubusercontent.com/20586368/235008929-9ae380a5-19e8-4281-b7de-3f346b94ba2b.jpg)
And when I try to Fix Match, then it shows up
![Screenshot  2023-04-27 190304](https://user-images.githubusercontent.com/20586368/235008968-b3758bef-8060-475b-956c-13ea974a0e8a.jpg)
Here's the log updated
[com.plexapp.agents.GayAdultScenes.log](https://github.com/CodyBerenson/PGMA-Modernized/files/11348546/com.plexapp.agents.GayAdultScenes.log)

About the Out of Athens file, it's named as (Falcon Studios) - Out of Athens 1 (2000)
![Screenshot 2023-04-27 190711](https://user-images.githubusercontent.com/20586368/235009216-5bb98991-aeb7-4687-92f9-e7030463ef26.jpg)

Thank you so much for your attention, Jason, really, it means too much =)

**Comment by JPH71** [comment]
Dear Dimitri,
There must be something you have missed...
I have tried the way big scene and as you can see from this picture - it has matched....
![image](https://user-images.githubusercontent.com/43102249/235031388-0f6343e6-b202-4795-869b-094ae0bfe35a.png)

This is how I have named it: (Lucas Entertainment) - Backdoor (Jonathan Agassi & Gabriel Lenfant) (2011).mp4
You can see this from the info screen
![image](https://user-images.githubusercontent.com/43102249/235031556-ef85d88d-3646-4c5c-bc54-96d27fdcddbb.png)

When it comes to Studio names - the matching is more lenient - you can put a space between Lucas and Entertainment... Drop stuff like .com, .net etc

You sent the wrong log file - so I still can not determine what went wrong.

As you are matching against waybig, send the waybig log

For out of athens - here is how it has been named....
![image](https://user-images.githubusercontent.com/43102249/235031915-00b5cfbb-186c-4370-9843-0681e09d1ec0.png)

Just thinking here - have your files got extensions.... I am just hazarding a guess....

When it comes to matching - try Refresh Metadata rather than Fix Match
![image](https://user-images.githubusercontent.com/43102249/235032136-3341b5c3-37e6-4be8-9b41-10f1e56cccad.png)


Hope it works - send me the log files if it does not work - and if it does work - let me know!!!

Cheers

Jason xxxx

**Comment by JPH71** [comment]
Also remember when sending pictures - avoid showing genitalia - or the group moderator gets in trouble.....

**Comment by dimmitri9** [comment]
Your Plex server looks beautiful!

Sending the log file from Way Big:
[com.plexapp.agents.WayBig.log](https://github.com/CodyBerenson/PGMA-Modernized/files/11349208/com.plexapp.agents.WayBig.log)

![Screenshot 2023-04-27 220424](https://user-images.githubusercontent.com/20586368/235036827-fa6086d6-d48d-456b-9b49-5a3ae0c4f7ec.jpg)
![Screenshot 2023-04-27 220510](https://user-images.githubusercontent.com/20586368/235036852-eefff4af-bdc2-44f9-9858-a19b2f7ec242.jpg)

Jason, I'm really trying to do my best to get it to work, I'm not a noob when to comes to pc but this... XD
Tried to Refresh metadata, and nothing and when I Fix Match it shows
![Screenshot 2023-04-27 220837](https://user-images.githubusercontent.com/20586368/235037202-07784ca2-3d56-4421-bfbf-5edd2be28148.jpg)

**Comment by CodyBerenson** [comment, marked as answer]
@JPH71 , @dimitrimahiro 

I took a quick look at Dimitri's log.  It looks like his Plex Server version is 1.32.0.6918, which we know does not work with any of our agents.  

@dimitrimahiro You probably need to downgrade your PMS to a previous version.  Please see discussion topic #232.

**Comment by JPH71** [comment]
Ah...
Bugger...

**Comment by JPH71** [comment]
Sorry Dimitri - I had completely forgotten how the latest version of plex
server refuses to scrape.....
I hope all is working now! And enjoy!

Jason
xxxx

**Comment by dimmitri9** [comment]
THANNKS SO MUCH GUYS!!!!!! =DDDDDDD

JASON, THANK YOU!!!! CODY, MAN YOU ROCK!!!!! =DDDD

After adding a bunch of scenes XD I came across a problem with some posts on WayBig

https://www.waybig.com/blog/2022/11/16/men-johnny-donovan-bottoms-for-troye-dean-cumpanion/
https://www.waybig.com/blog/2022/09/26/men-malik-delgaty-fucks-paul-wagner-warm-welcum/
https://www.waybig.com/blog/2022/11/01/men-olivier-robert-felix-fox-and-tayler-tash-cockrise-yoga/

I'm changing MEN.Com to MEN then to Men, then to Men.com, but nothing match them
Then changin the ´´ to '' but again, nothing

Here are the files named right now
![Screenshot 2023-04-29 123617](https://user-images.githubusercontent.com/20586368/235313602-42c25f5f-65e2-4f17-bffb-3bd421470076.jpg)

![Screenshot 2023-04-29 123816](https://user-images.githubusercontent.com/20586368/235313688-e5ef5810-7166-4c57-9221-070a771213df.jpg)

**Reply by CodyBerenson** [reply]
@dimitrimahiro Jason has coded an update to address certain issues with titles with apostrophes in them. I will be posting the new release to the repository in a couple hours. I believe this code may help you.

**Reply by dimmitri9** [reply]
Wow, I'm amazed!!!!! =) will it be needed to update something...

**Reply by CodyBerenson** [reply]
The updated code has been posted.  You can download, remove the previous plugins, and replace with the updated code.  This is from our our FAQ:

Q: Does the code for the Agents ever change/get updated?
A: Yes! We are always developing new agents. For existing agents, new functionality may be added (e.g., language translation support), or site updates to an index may break the Agent's ability to search. It is a good practice to periodically check back and install the latest set of 
code. Remember to stop and restart your Plex Media Server (PMS) in order for the new/updated Agents to take effect.
